pkg/server/datastore/sqlstore/models.go

```diff
@@ -162,6 +162,30 @@ func (FederatedTrustDomain) TableName() string {
 	return "federated_trust_domains"
 }
 
+// EntryEvent holds the entry id of an entry that had an event
+type EntryEvent struct {
+	Model
+
+	EntryID string `gorm:"unique_index"`
+}
+
+// TableName gets table name for Entry Events
+func (EntryEvent) TableName() string {
+	return "entry_events"
```

Comment:
NIT: since entry table is called `registered_entries` may we set table name `registered_entries_events`? (similar comment to `attested_node_entries`)

Reply:
> NIT: since entry table is called `registered_entries` may we set table name `registered_entries_events`? (similar comment to `attested_node_entries`)

It was done many commits back, which was a bit long in various places.  That led to a shortening suggestion that turned it into "events" which was then too short to be descriptive.  "entries_events" should be fine in my opinion, even if the other item is called "registered_entries" because we don't have any other kind of entries.

Personally, I'd love to see "registered" dropped from entries.  It's like saying "stored_entries" when there's no place to store "not stored entries" :)

Reply:
Though it is a bit long, I prefer the `registered_entries_events` as it makes it clear what its attached to.

Reply:
Not a point that I'm against, go for it!

Reply:
names updated.